src/Databricks/help/Az.Databricks.md

```diff
@@ -10,6 +10,13 @@ Locale: en-US
 ## Description
 Microsoft Azure PowerShell: Databricks cmdlets
 
+This module is still in preview and must be installed separately using the following command
```

Comment:
What does it need to be installed separately from? I used the following message in the [quickstart for Databricks](https://docs.microsoft.com/en-us/azure/azure-databricks/quickstart-create-databricks-workspace-powershell) that I wrote:

_"While the Az.Databricks PowerShell module is in preview, you must install it separately from the Az PowerShell module using the following command: `Install-Module -Name Az.Databricks -AllowPrerelease`. Once the Az.Databricks PowerShell module is generally available, it becomes part of future Az PowerShell module releases and available natively from within Azure Cloud Shell."_

Reply:
I just copy/past your content, except `-AllowPrerelease` is not required according to my test.